source/github-helpers/index.ts

```diff
@@ -106,6 +109,14 @@ export function preventPrCommitLinkLoss(url: string, pr: string, commit: string,
 
 	return `[\`${commit}\` (#${pr})](${url})`;
 }
+//To be  used as replacer callback in string.replace() for compare links
+export function preventPrCompareLinkLoss(url: string, compareFile1: string, compareFile2: string, compare: string, index: number, fullText: string): string {
+	if (fullText[index + url.length] === ')') {
+		return url;
+	}
+
+	return `[\`${compare}\` (${compareFile1}-${compareFile2})](${url})`;
```

Comment:
We can capture the whole `#diff` part and just append it. This is not a commit hash and it's meaningless.
```suggestion
	return `[\`${compare}\`${diff}](${url})`;
```

It would appear as: [`a...b`#diff-erxvgbhjnm](#)